client/js/utils/api-utils.js

```diff
@@ -1,18 +1,17 @@
 var axios = require('axios');
-var weekData = require('../weekData');
 var server = 'http://localhost:8000';
 
 var ApiUtils = {
 
-  addEvent: function (event, callback) {
-    // this is currently mocking server side data manipulation
-    // NOT the final implementation
-    if (weekData.hasOwnProperty(event.date)) {
-      weekData[event.date].push(event);
-    } else {
-      weekData[event.date] = [event];
-    }
-    callback(weekData);
+  addGig: function (gig, callback) {
+    var path = server + '/api/organizations/1/gigs/';
+    axios.post(path, {
```

Comment:
no need to use 'params' here, can just do: 

    axios.post(path, {
      gig: gig
    }).....